src/client/components/pages/entities/identifiers.js

```diff
@@ -43,14 +46,17 @@ function EntityIdentifiers({identifiers, identifierTypes}) {
 									</dd>
 								)
 							);
-					if (!identifierValues.length) {
-						return null;
-					}
-					return [
-						<dt key={type.id}>{type.label}</dt>,
-						identifierValues
-					];
-				})
+						if (!identifierValues.length) {
+							return null;
+						}
+						return [
+							<dt key={type.id}>{type.label}</dt>,
+							identifierValues
+						];
+					}) :
+					<p className="text-muted">
+						<b>No identifiers found.</b> Click the <a href={editLink}>&quot;Edit&quot;</a> button to create a new identifier (e.g. ISBN, Wikidata ID).
```

Comment:
I think maybe this would be shorter:

```suggestion
						<b>No identifiers.</b> <a href={editLink}>Click here to edit</a> and add new identifiers (e.g. ISBN, Wikidata ID, etc.).
```